src/style/app.module.css

```diff
@@ -7251,3 +7251,48 @@ button[data-testid='createPostBtn'] {
   flex-direction: row;
   justify-content: center;
 }
+
+.alignimg {
+  border-radius: 50%;
+  background-color: var(--white-color);
```

Comment:
Use variable names that describe the function of the color and not the color itself. This will help with adding a Dark Mode feature.
- For example `--search-button-bg` is much better than `--light-blue`